core/gatsby-theme-docz/src/node/onPreInit.js

```diff
@@ -17,7 +17,7 @@ const touchTemplateWithPaths = paths => async (filepath, opts) => {
   await touch(dest, raw)
 }
 
-module.exports = async opts => {
+module.exports = async (_ opts) => {
```

Comment:
Shouldn't this be `async (_, opts) => {` instead?

Missing the `,` (comma / period).

Reply:
Yes, yes it should. Fixed!